Source/Bot/Bot.swift

```diff
@@ -20,6 +20,7 @@ typealias PaginatedCompletion = ((PaginatedKeyValueDataProxy, Error?) -> Void)
 typealias HashtagCompletion = ((Hashtag?, Error?) -> Void)
 typealias HashtagsCompletion = (([Hashtag], Error?) -> Void)
 typealias PublishCompletion = ((MessageIdentifier, Error?) -> Void)
+typealias CountCompletion = ((Int, Error?) -> Void)
```

Comment:
No need to change this now if you don't want but in general I think it's better to use the `Result` type for completion handlers that return a value or an error.

Reply:
I also like using Result, I will change it